setup.py

```diff
@@ -296,16 +297,20 @@ def validate_and_update_archs(archs):
         torch._C._GLIBCXX_USE_CXX11_ABI = True
 
     fa_sources = ["csrc/flash_attn_ck/flash_api.cpp",
```

Comment:
could we have better name for the `fa_source` & `sources`? I suggest to use `sources` & `renamed_sources`

Reply:
f41af536ab05533e80c7b13d67a9262f4d967087